sigh/lib/sigh/runner.rb

```diff
@@ -65,7 +65,7 @@ def profile_type
     # Fetches a profile matching the user's search requirements
     def fetch_profiles
       UI.message "Fetching profiles..."
-      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier]).find_all(&:valid?)
+      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier])
```

Comment:
`sigh` can't repair the profile if it never finds it in the first place
we could double check that its valid before we download it at the end?